libs/pipelines/api_pipeline.py

```diff
@@ -59,18 +65,35 @@ def timeseries(self) -> OneRegionTimeseriesDataset:
 
     @staticmethod
     def from_region_and_intervention(
-        region: pipeline.Region, intervention: Intervention, model_output_dir: pathlib.Path
+        region: pipeline.Region,
+        intervention: Intervention,
+        model_output_dir: pathlib.Path,
+        rt_data: MultiRegionTimeseriesDataset,
```

Comment:
Change to Optional[MultiRegionTimeseriesDataset] because it can be None. Ideally some kind of type checking would catch this.

Reply:
I think the multi region is never None